src/pages/Login/CustomInput/CustomInput.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const CustomInput = ({ ariaLabel, type, placeholder, styleInput, onChange, value }) => {
```

Comment:
Instead of having a long line of props, please have each one of them in a separate line.
Make sure to maintain the indentation when you do that.